package.json

```diff
@@ -4,12 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run unit && npm run cover && npm run snyk",
+    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage && npm run snyk",
     "unit": "mocha --recursive test/spec/ --require test/helper --timeout 300000",
     "cover": "istanbul cover _mocha -- -R dot --recursive test/spec/ --require test/helper --timeout 300000",
-    "check-coverage": "istanbul check-coverage --statement 98.31 --branch 94.89 --function 96.30 --line 98.31",
+    "check-coverage": "istanbul check-coverage --statement 90 --branch 90 --function 90 --line 90",
     "lint": "eslint .",
-    "snyk": "snyk test",
+    "snyk": "snyk auth $SNYK_TOKEN && snyk test",
```

Comment:
This needs to be `$SNYK_AUTH` not `$SNYK_TOKEN` judging by the current travis settings.

<img width="569" alt="screen shot 2017-02-10 at 17 32 19" src="https://cloud.githubusercontent.com/assets/117398/22837060/ebea1c88-efb6-11e6-9cca-e79243d57e49.png">